shared/constants/types/fs.js

```diff
@@ -55,6 +56,7 @@ export type _FolderPathItem = {
   children: I.Set<string>,
   favoriteChildren: I.Set<string>,
   resetParticipants: Array<string>,
+  teamID?: RPCTypes.TeamID,
```

Comment:
Can this go into tlfMeta?

Also I wonder if we should have a Types.TlfPathItem so that we don't have to put TLF metadata into normal Folders.

Reply:
Yeah, good call.

Reply:
Oh actually, I thought of doing that but it complicates the `listFavorites` result coming back, since that just drop-in replaces `tlfMeta`. I'd have to selectively replace parts of `tlfMeta` otherwise.

Reply:
`TlfPathItem` is a great idea but it'll be a headache to make work.

Reply:
Yeah that is a refactor that should go in its own PR.

Reply:
Sounds good. When we are done with these changes, I'll try to do that refactor (so that it doesn't cause conflicts)

Reply:
I can do it, we just need to plan it separately.